app/views/hubstats/partials/_deploy.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row single-pull">
```

Comment:
You should make a new HTML class for these to support varying styles between pulls and deploys (even if they are the same right now).